include/evmmax/evmmax.hpp

```diff
@@ -8,20 +8,23 @@
 namespace evmmax
 {
 
-/// Compute the modulus inverse for Montgomery multiplication, i.e. N': mod⋅N' = 2⁶⁴-1.
+/// Compute the modulus inverse for Montgomery multiplication, i.e., N': mod⋅N' = 2⁶⁴-1.
 ///
 /// @param mod0  The least significant word of the modulus.
 constexpr uint64_t compute_mod_inv(uint64_t mod0) noexcept
```

Comment:
Why `mod0`?

Reply:
The least significant word of `mod`. We can change this to something generic because this context belongs to the user, not here.